pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -500,7 +508,26 @@ async def find_bugs(self, kind: str, permissive: bool = False) -> Optional[Issue
                 issues = Issues(fixed=[Issue(id=b, source="issues.redhat.com") for b in bugs])
                 self._issues_by_kind[advisory_kind] = issues
 
-        return self._issues_by_kind.get(kind)
+        shipment.shipment.data.releaseNotes.issues = self._issues_by_kind.get(kind)
+
+    async def attach_cve_flaws(self, kind, shipment):
+        with TemporaryDirectory() as elliott_working:
+            # Replace the elliott working dir to allow concurrent commands execution
+            attach_cve_flaws_command = [
+                arg if not arg.startswith('--working-dir=') else f'--working-dir={elliott_working}'
+                for arg in self._elliott_base_command
+            ]
+            attach_cve_flaws_command += ['attach-cve-flaws', f'--use-default-advisory={kind}', '--output=json']
+
+            _LOGGER.info('Running elliott attach-cve-flaws...')
+            rc, stdout, stderr = await exectools.cmd_gather_async(attach_cve_flaws_command, stderr=None)
+            if stdout:
+                _LOGGER.info("Shipment find bugs command stdout:\n %s", stdout)
+            if rc != 0:
```

Comment:
I think this is the same as other places in the pipeline.. but now that i read cmd_gather_async, it has check=True, so this check might be a redundant

Reply:
yes, check=True by default. I replicated the pattern from the same file, changed all occurrences with be66e85a04c53fdf7321c100f4cac9e8b97b9cc1